### PR TITLE
Update Update-AzSynapseSparkPool.md

### PR DESCRIPTION
**Pull request by kevinjaku**
## Description

Adding notes wrt Synapse Spark version using Update-AzSynapseSparkPool cmdlet based on updates in https://learn.microsoft.com/en-ca/azure/synapse-analytics/spark/apache-spark-version-support#migration-between-apache-spark-versions---support

## Mandatory Checklist

- Please choose the target release of Azure PowerShell
  - [x] [General release](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Public preview](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Private preview](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Engineering build](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] N/A

- [x] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For SDK-based development mode, update `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`.
        * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. 
    * For autorest-based development mode, include the changelog in the PR description.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

**Comment by kevinjaku** [comment]
@msJinLei could you please help us review this PR? @wonner has been added as a reviewer too but she had some issues with her Gtihub account, so couldn't complete the approval

File: src/Synapse/Synapse/help/Update-AzSynapseSparkPool.md [hunk 1]
```diff
@@ -69,6 +69,9 @@ Update-AzSynapseSparkPool -ResourceId <String> [-Tag <Hashtable>] [-EnableAutoSc
 ## DESCRIPTION
 The **Update-AzSynapseSparkPool** cmdlet updates an Apache Spark pool in Azure Synapse Analytics.
 
+> [!NOTE]
+> If the ```-SparkVersion``` parameter is used to upgrade the Synapse Spark runtime version, ensure that the Spark pool doesn't have any attached custom libraries or packages. Refer to [Migration between Apache Spark versions](/azure/synapse-analytics/spark/apache-spark-version-support.md#migration-between-apache-spark-versions---support) for more details.
```
Comment (wyunchi-ms) [inline src/Synapse/Synapse/help/Update-AzSynapseSparkPool.md:73]: Please use absolute path of the url.

Comment (wyunchi-ms) [inline src/Synapse/Synapse/help/Update-AzSynapseSparkPool.md:73]:
```suggestion
> If the `-SparkVersion` parameter is used to upgrade the Synapse Spark runtime version, ensure that the Spark pool doesn't have any attached custom libraries or packages. Refer to [Migration between Apache Spark versions](/azure/synapse-analytics/spark/apache-spark-version-support.md#migration-between-apache-spark-versions---support) for more details.
```

**Review by wyunchi-ms — APPROVED** [review]

**Pull request merged**